run.py

```diff
@@ -744,11 +762,28 @@ def build_env(self):
                 env[key] = str(self.config["env"][key])
         return env
 
+    def wait_suricata_start(self, p):
+        # timeout wait 2 second at most for Suricata to be ready
```

Comment:
2 secs seems really low, I fear things will break on busy builders or with slightly larger rulesets slowing down the start up